### DISCUSSION TITLE
DCP Camera Profile Support

### DISCUSSION THREAD

**Discussion by kamikaze** [colour-science/colour#1126] [Q&A]
### Question

Hi. I'm trying to apply camera profile to a raw image (Sony ARW, reading with rawpy). Can't figure out if colour is capable of doing this in any possible way. Is there such support? Thanks

**Comment by thomasmansencal** [comment]
Hi @kamikaze,

Colour does not support DCP profile support. [colour-hdri](https://github.com/colour-science/colour-hdri) is able to reproduce the DNG SDK white balancing but does not support DCP profiles either. That being said, depending on what you are trying to achieve, there are ways to extract useful information out of them, and then use it for processing.

Cheers,

Thomas
